scheduler/scheduler.go

```diff
@@ -36,6 +36,7 @@ import (
 	"github.com/intelsdi-x/snap/scheduler/wmap"
 )
 
+// Schedule var variables
```

Comment:
You could describe how HandlerRegistrationName is used to register a gomit event handler.